### ISSUE TITLE
Proposal: Migrate this repository to Kialo

### ISSUE THREAD

**Issue by kendfrey** [JavaScriptRoom/culture#49]
[Kialo](https://www.kialo.com) is a discussion site where discussions are based on a tree of claims. Each claim can contain sub-claims that support or oppose it. I've formatted this issue like a Kialo discussion so you can see how it is organized. Each line represents one claim, and indenting represents sub-claims.

:grey_question: The JS room discussion should take place on Kialo, rather than GitHub.

* :heavy_check_mark: Kialo is designed so that each argument or claim can be discussed by itself.
  * :heavy_check_mark: This prevents arguments about different points from ending up in the same thread, and making it hard to follow.
  * :heavy_check_mark: Claims can be linked as pros or cons for multiple parent claims, as this one is.
* :heavy_check_mark: Kialo has better tools than GitHub for managing discussions and arguments.
  * :heavy_check_mark: You can vote on individual claims and their pros and cons.
  * :heavy_check_mark: Claims can be linked as pros or cons for multiple parent claims, as this one is.
  * :heavy_check_mark: Poorly presented claims can be flagged for improvement or removal.
- :x: Most people in the JS room have GitHub accounts, but do not have Kialo accounts.
- :x: Kialo discussions are not editable by the public.
  * :heavy_check_mark: Non-ROs would require a RO to approve any issues they wish to discuss.
  * :heavy_check_mark: JS room discussions would need to be private (invite-only) anyway.
  - :x: This prevents trolls and unhelpful discussions.
- :x: Kialo is beta software. There are minor usability issues and bugs, as well as frequent patches.
- :x: Kialo doesn't do as well with opinion-based discussion.
- :x: The moderation repo wouldn't fit well on Kialo.
  * :heavy_check_mark: Moderation issues are typically more opinion-based.
    * :heavy_check_mark: Kialo doesn't do as well with opinion-based discussion.
  - :x: Kialo has visibility settings that allow discussions to be private.

I doubt moving from GitHub is the right thing to do, but I'm suggesting this here to hear your thoughts. This is mainly driven by the desire to avoid unpleasant back-and-forth arguing by replacing it with something more structured.

**Comment by FirstWhack** [comment]
In my limited experience Kialo is very convoluted and would not be an efficient platform for discussions including (or even started by) randoms.

This is the random repo, it really is hardly even used. I vote just leave it.

**Comment by luggage66** [comment]
I wouldn't really be opposed to trying it if others did, but I feel this is better on the grounds that we want a 'lowest common denominator' and Github is that for our demographic.

**Comment by benjamingr** [comment]
Kialo looks nice.

The reason I like GitHub better is because everyone already has it and knows how to use it - not because I think it's a great platform for this sort of thing.

**Comment by ssube** [comment]
Already having github is nice, but it's not a great platform for comment threads. If Kialo is easy to set up and SSO in, I'll try it, but it'll be hard to test without some issues we need to address.

**Comment by Trasiva** [comment]
I'm up for trying it, but it doesn't sound like it's a perfect fit for us either.

**Comment by Zirak** [comment]
It sounds cool but

* ❌ Most of what we do is have discussions, not logical debates
* ❌ (Unless I'm mistaken) We don't find github a difficult platform to consider migrating from
* ❌ Kialo is a weird name
  * ✔️ It totes is
  * ❌ Github s also a weird name

* ❔ It's convenient to host discussions in issues as it's also a form of documentation - we can link to comments and discussions all harboured under the same repo umbrella. Is there such an equivalent in Kialo? A "js room" general discussion branching into sub-discussions?

**Comment by GNi33** [comment]
It is an interesting platform. That being said, I really doubt that it will be a better platform than github for our (most common) usecase. Comments there will need to be very specific replies to each point, which either will result in multiple comments needed to be made or some points being lost in the comment tree. 

I could see it being a somewhat good fit for certain discussions, but not for the majority of things discussed here.

**Comment by kendfrey** [comment]
@Zirak There's a team feature (basically like GitHub) so we could make a JS room team, and that team would be used for discussions. I'm not sure if discussions can be owned by teams, or just people. Also it is possible to cross-reference discussions, so in theory there could be a master discussion with links to the other discussions.

**Comment by benjamingr** [comment]
Closing after ~2 days of inactivity. Feel free to reopen if you think there is more discussion to be had.

**Comment by kendfrey** [comment]
Honestly, the response was more positive then I expected, but it doesn't look like it's a good fit, so we'll go on as before.
